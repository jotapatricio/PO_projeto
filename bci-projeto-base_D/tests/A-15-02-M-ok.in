6
4
Joao Fonseca
4
Carlos Fonseca
4
Fernando Fonseca2
0
0
