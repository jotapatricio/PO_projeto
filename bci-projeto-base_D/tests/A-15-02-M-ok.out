Menu Principal
1 - Abrir ficheiro
2 - Guardar ficheiro
3 - Ver data
4 - Avançar data
5 - Menu de gestão de utentes
6 - Menu de gestão de obras
7 - Menu de gestão de requisições
0 - Sair
Escolha uma opção: Menu de obras
1 - Mostrar obra
2 - Mostrar todas as obras
3 - Efectuar pesquisa de termos
4 - Mostrar todas as obras de um criador
5 - Alterar inventário de uma obra
0 - Sair
Escolha uma opção: Introduza o identificador do criador: 1 - 2 de 2 - Livro - A arte de sobreviver no 36 - 20 - Ficção - Carlos Fonseca2; Fernando Fonseca2; Joao Fonseca - 1234567892
Menu de obras
1 - Mostrar obra
2 - Mostrar todas as obras
3 - Efectuar pesquisa de termos
4 - Mostrar todas as obras de um criador
5 - Alterar inventário de uma obra
0 - Sair
Escolha uma opção: Introduza o identificador do criador: 2 - 3 de 3 - Livro - Analise Matematica sem Mestre - 19 - Técnica e Científica - Carlos Fonseca - 1234567890
Menu de obras
1 - Mostrar obra
2 - Mostrar todas as obras
3 - Efectuar pesquisa de termos
4 - Mostrar todas as obras de um criador
5 - Alterar inventário de uma obra
0 - Sair
Escolha uma opção: Introduza o identificador do criador: 1 - 2 de 2 - Livro - A arte de sobreviver no 36 - 20 - Ficção - Carlos Fonseca2; Fernando Fonseca2; Joao Fonseca - 1234567892
Menu de obras
1 - Mostrar obra
2 - Mostrar todas as obras
3 - Efectuar pesquisa de termos
4 - Mostrar todas as obras de um criador
5 - Alterar inventário de uma obra
0 - Sair
Escolha uma opção: Menu Principal
1 - Abrir ficheiro
2 - Guardar ficheiro
3 - Ver data
4 - Avançar data
5 - Menu de gestão de utentes
6 - Menu de gestão de obras
7 - Menu de gestão de requisições
0 - Sair
Escolha uma opção: 